lib/util/exec/containerbase.ts

```diff
@@ -186,6 +186,11 @@ const allToolConfig: Record<string, ToolConfig> = {
     packageName: 'rust',
     versioning: semverVersioningId,
   },
+  uv: {
+    datasource: 'github-releases',
+    packageName: 'astral-sh/uv',
+    versioning: semverVersioningId,
```

Comment:
uv is also available [on PyPI](https://pypi.org/project/uv/). Should we retrieve it from there, or is it better to use GitHub? In this file, PDM is also retrieved from GitHub releases, but Poetry is retrieved from PyPI, although it also has GitHub releases.

Reply:
use pypi